File: OCDemo_v6.5.xx/iOSDemo/Others/Controller/sub/BaseNormalBarVC.m
//
//  BaseNormalBarVC.m
//  AnyThinkSDKDemo
//
//  Created by TopOn技术支持 on 2025/7/3.
//

#import "BaseNormalBarVC.h"

@interface BaseNormalBarVC ()

@end

@implementation BaseNormalBarVC

#pragma mark - Demo Needed (No need to integrate)
- (void)viewDidLoad {
    [super viewDidLoad];
     
    [self setupDemoUI];
}

- (void)setupDemoUI {
    [self addNormalBarWithTitle:self.title];
    [self addLogTextView];
    [self addFootView];
}

@end

File: SwiftDemo/SwiftDemo/Others/Controller/sub/BaseNormalBarVC.swift
//
//  BaseNormalBarVC.swift
//  SwiftDemo
//
//  Created by ltz on 2025/8/20.
//

import UIKit

class BaseNormalBarVC: BaseVC {
    
    // MARK: - Demo Needed (No need to integrate)
    override func viewDidLoad() {
        super.viewDidLoad()
        
        setupDemoUI()
    }
    
    func setupDemoUI() {
        addNormalBar(withTitle: title)
        addLogTextView()
        addFootView()
    }
}
